libraries/pbrlib/genglsl/mx_sheen_brdf.glsl

```diff
@@ -8,6 +8,12 @@ void mx_sheen_brdf_reflection(vec3 L, vec3 V, float weight, vec3 color, float ro
         return;
     }
 
+    // Make sure we are forward facing.
+    if (dot(N,V) < 0.0)
```

Comment:
I kind of remember from previous history that you may want to put a numeric epsilon tolerance on this.